.gitignore

```diff
@@ -3,6 +3,7 @@
 .bundle
 .config
 .yardoc
+.idea/
```

Comment:
Since this is very specific to your development setup, I'd ask you to revert that change. Suggestion: you can specify a global gitignore file in your global git config (`~/.gitconfig`) like so:

```
[core]
excludesfile = ~/.gitignore
```